.github/workflows/parallel_build.yaml

```
@@ -82,6 +82,31 @@ jobs:
         echo "IDF_BRANCH=$IDF_BRANCH"
         echo "IDF_COMMIT=$IDF_COMMIT"
         idf_version_string="${IDF_BRANCH//\//_}-$IDF_COMMIT"
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)